pkg/apis/logging/v1/clusterlogging_types.go

```diff
@@ -38,6 +38,11 @@ type ClusterLoggingSpec struct {
 	//
 	// +nullable
 	Curation *CurationSpec `json:"curation,omitempty"`
+
+	// Specification for output tuning parameters
+	//
+	// +nullable
+	Output *OutputConfSpec `json:"output,omitempty"`
```

Comment:
shouldn't this be nested under the fluentd spec?

Reply:
as we expand other collectors we would nest specific output specs under those as well -- also we may not need to do that for other collectors